go

```diff
@@ -46,4 +46,15 @@ def_command :lint, 'Run style-checking tools' do |files|
   lint_javascript Dir.pwd, files['.js'] # uses node_modules/jshint
 end
 
+def_command :update_cg_style, 'Update cloudgov_styles' do
+  vendor_dir = './public/vendor'
+  exec_cmd "npm install"
+  exec_cmd "rm -rf #{vendor_dir}"
+  exec_cmd "mkdir #{vendor_dir}"
+  %w(js css img).each do |val|
+    exec_cmd "cp -R ./node_modules/cloudgov-style/#{val} #{vendor_dir}/"
+  end
+  exec_cmd "cp -R ./node_modules/cloudgov-style/font/ #{vendor_dir}/fonts"
```

Comment:
FYI that you can use an [`npmrc`](https://docs.npmjs.com/files/npmrc#per-project-config-file) file with `prefix = public/vendor`, which should install the files to the right place without needing to copy them around.

https://docs.npmjs.com/misc/config#prefix

/cc